files/grid-studies-2-1.py

```diff
@@ -46,5 +57,10 @@ def serialosc_device_added(id, type, port):
     serialosc = monome.SerialOsc()
     serialosc.device_added_event.add_handler(serialosc_device_added)
 
-    loop.run_until_complete(serialosc.connect())
-    loop.run_forever()
+    play_task = asyncio.create_task(grid_studies.play())
```

Comment:
it's also possible to start this in app constructor too:

```
class GridStudies(monome.GridApp):
    def __init__(self):
        super().__init__()

        ...

        self.play_task = asyncio.ensure_future(self.play()) # it doesn't matter if ensure_future or create_task is used here
```

Reply:
oooo rad, so much clearer to have the task build up in the init!